pkgs/servers/bge/default.nix

```diff
@@ -0,0 +1,54 @@
+{sbt,stdenv, fetchFromGitHub, jdk, makeWrapper}:
+
+with stdenv.lib;
+stdenv.mkDerivation {
+  name = "bge-3.1";
+
+  meta = {
+    description = "A blockchain analysis server";
+    longDescription = "Bitcoin Graph Explorer is a live bitcoin blockchain analysis server including a REST API. It is written in Scala.";
+    homepage = "https://bitcoinprivacy.net";
+    maintainers =  with maintainers; [ jorgemartinezpizarro stefanwouldgo ];
```

Comment:
@jorgemartinezpizarro, could you please confirm you want to maintain the Nix package?